webpack.config.js

```diff
@@ -233,7 +236,7 @@ let appConfig = {
         exclude: /(vendor|node_modules|dist)/,
         // Make sure we typecheck in CI, but not for local dev since that is run with
         // the fork-ts plugin
-        use: !IS_CI ? babelLoaderConfig : [babelLoaderConfig, tsLoaderConfig],
+        use: IS_DEV_ENV ? [babelLoaderConfig, tsLoaderConfig] : babelLoaderConfig,
```

Comment:
Now this becomes as follows, based on my suggestion above:
```suggestion
        use: SHOULD_FORK_TS ? babelLoaderConfig : [babelLoaderConfig, tsLoaderConfig],
```